src/fx-data-download.py

```diff
@@ -353,11 +404,15 @@ def bt5_to_csv(self):
                 data = f.read()
         # Workaround for liblzma bug (https://bugs.python.org/issue21872)
         except EOFError:
-            print("Info: Ran into liblzma decompressor bug, falling back to command line decompression...")
+            print('"Info: Ran into liblzma decompressor bug,'
+                  'falling back to command line decompression..."')
             try:
-                pipe = subprocess.Popen(['xz', '-dc', self.path], stdout=subprocess.PIPE)
+                pipe = subprocess.Popen(['xz',
+                                         '-dc',
+                                         self.path], stdout=subprocess.PIPE)
             except FileNotFoundError:
-                print("Error: Unable to find the 'xz' LZMA decompressor utility in your PATH, moving on.")
+                print('"Error: Unable to find the 'xz' LZMA'
```

Comment:
Multiline should go like:
```
"I am learning Python." \
"I love Software Testing Material." \
"Cool Python Tips."
```

No need to add extra apostrophes.

Reply:
This line breaks the test, see: : https://github.com/FX31337/FX-Data-Download-Action/pull/3/checks?check_run_id=2045693087#step:3:82
As you're mixing different quotes and apostrophes.